Doc/source/designspaceLib/readme.rst

```diff
@@ -846,15 +844,28 @@ Example
 -  Container for ``rule`` elements
 -  The rules are evaluated in this order.
 
+Rules describe designspace areas in which one glyph should be replaced by another.
+A rule has a name and a number of conditionsets. The rule also contains a list of
+glyphname pairs: the glyphs that need to be substituted. For a rule to be triggered
+**only one** of the conditionsets needs to be true, ``OR``. Within a conditionset 
+**all** conditions need to be true, ``AND``.
+
+
+Attributes
+----------
+
+-  ``processing``: flag, optional. Valid values are [``first``, ``last``]. This flag indicates whether the substitution rules should be applied before or after other glyph substitution features.
+-  If no ``processing`` attribute is given, interpret as ``first``.
```

Comment:
in the other thread @twardoch raised a valid point about the undefined meaning of processing=post when generating instance UFOs from a designspace. Should the spec say something here?

Reply:
The "UFO instances" section about rules currently doesn't say anything about other glyph substitutions, so I think a general clarification of how to deal with that would be good.

Reply:
An important detail is (under 5.0 rules element) "The rules are evaluated in this order". This is currently not true with varLib.featureVars, and I'm not sure it should be. The order is only relevant if rules react on each other (eg. rule N+1 acts on the output of rule N).

While this may be fixable (generate substitution lookups in the order of the original rules), I'm not sure what the use case is.

Reply:
(Perhaps this discussion deserves its own issue.)